introduction.md

```diff
@@ -1,2 +1,2 @@
-Hello, i am Oleg Laktionov.
-My favorite path is JavaScript.
+* JavaScript
+* Ruby
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?